scripts/afterRelease.sh

```diff
@@ -0,0 +1,24 @@
+brew install hub
+yarn typedoc --json docs.json
+node scripts/postProcessDocs.js
+
+packageVersion=$(node -e "const fs = require('fs');\
```

Comment:
Replace this with `packageVersion=${jq .version ./package.json}`?

Reply:
nice